es/mappings/make.py

```diff
@@ -219,6 +219,10 @@ def IsCyrillic(lang, something):
         "result_type": {
           "type": "keyword",
         },
+        # Document index date.
+        "index_date": {
+          "type": "date",
```

Comment:
Why not:
"format": "strict_date",
As in effective_date?

Reply:
I think we should use here:
"epoch_millis"

Reply:
done